src/Components/NavBar/__tests__/NavBar.test.tsx

```diff
@@ -51,8 +51,8 @@ describe("NavBar", () => {
 
   describe("desktop", () => {
     const defaultLinks = [
-      ["/collect", "Artworks"],
-      ["/artists", "Artists"],
+      [undefined, "ArtworksReveal more"],
+      [undefined, "ArtistsReveal more"],
```

Comment:
Two things regarding this test: 
1) For the urls, Artworks and Artists no longer have  a`href`. 
2) The "Reveal more" gets appended on because of the [`ChevronIcon` component's](https://github.com/artsy/palette/blob/d4acee6bbdf69269b0f6ec12702c40f4608345cb/packages/palette/src/svgs/ArrowDownIcon.tsx#L7) `<Title>` . I considered conditionally rendering that in palette but wondered if we had that there for SEO purposes. I'm open to a better way of handling that for this test.

Reply:
These changes make sense to me-- I think it's fine to keep the `Title` text there!

Eventually I think we'll add back the `href`s on those items, so this sets us up well for that future.